mmv1/templates/terraform/examples/vertex_ai_featureonlinestore_featureview_cross_project.tf.erb

```diff
@@ -0,0 +1,132 @@
+resource "google_project" "project" {
```

Comment:
Do you mind explaining in writing which resource(s) should be in project A (the standard test project) and project B (the project we're creating inline in the test)? I'm not familiar with the CUJ we're covering, so I'm just poking at things (like with my comment below) otherwise.

Reply:
In the project B (inline created project), we create the BQ dataset & table, FeatureGroup and Feature. In project A, we create the FeatureOnlineStore and FeatureView. In the FeatureView, we point to the FeatureGroup and Features created in ProjectB as the source along with the project B's project_number. This is the usecase. 

Previously cross project FeatureGroup referencing wasn't allowed. We're trying to enable that in terraform with this change.

Reply:
Thanks!